editor/src/components/inspector/sections/component-section/component-section.spec.browser2.tsx

```diff
@@ -968,52 +970,6 @@ describe('Controls from registering components', () => {
   })
 })
 
-/*
-describe('Delete cartouche handling', () => {
```

Comment:
Please don't remove the commented out tests. Please can you instead try restoring them, since they _should_ work now with this new approach. They were commented because the old approach was causing unrelated tests to fail when these ran